diff -Naur arc-firefox-theme-52.20170311.orig/arc-firefox-theme/chrome.manifest arc-firefox-theme-52.20170311/arc-firefox-theme/chrome.manifest
--- arc-firefox-theme-52.20170311.orig/arc-firefox-theme/chrome.manifest	2017-03-11 08:55:24.000000000 -0500
+++ arc-firefox-theme-52.20170311/arc-firefox-theme/chrome.manifest	2017-03-22 22:26:03.992065283 -0500
@@ -1,4 +1,4 @@
-skin    browser         arc-theme   chrome/browser/
-skin    communicator    arc-theme   chrome/communicator/
-skin    global          arc-theme   chrome/global/
-skin    mozapps         arc-theme   chrome/mozapps/
+skin    browser         deskos-theme   chrome/browser/
+skin    communicator    deskos-theme   chrome/communicator/
+skin    global          deskos-theme   chrome/global/
+skin    mozapps         deskos-theme   chrome/mozapps/
diff -Naur arc-firefox-theme-52.20170311.orig/arc-firefox-theme/install.rdf arc-firefox-theme-52.20170311/arc-firefox-theme/install.rdf
--- arc-firefox-theme-52.20170311.orig/arc-firefox-theme/install.rdf	2017-03-11 08:55:24.000000000 -0500
+++ arc-firefox-theme-52.20170311/arc-firefox-theme/install.rdf	2017-03-22 22:37:00.084265270 -0500
@@ -7,10 +7,10 @@
     <em:id>{52c2877e-44e1-11e5-8874-a62d1d5d46B0}</em:id>
     <em:version>52.20170311</em:version>
     <em:type>4</em:type>
-    <em:name>Arc Theme</em:name>
-    <em:description>Arc Theme for Firefox</em:description>
+    <em:name>DeskOS Theme</em:name>
+    <em:description>DeskOS Theme for Firefox</em:description>
     <em:creator>horst3180</em:creator>
-    <em:internalName>arc-theme</em:internalName>
+    <em:internalName>deskos-theme</em:internalName>
     <em:skinnable>false</em:skinnable>
     <em:multiprocessCompatible>true</em:multiprocessCompatible>
     <em:targetPlatform>Linux</em:targetPlatform>
